components/landing/topCollections/TopCollectionsGrid.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <DynamicGrid
```

Comment:
Is it possible to use css instead? demo here https://play.tailwindcss.com/HY9KzaQxiJ?layout=preview&size=1750x720

Reply:
but this does not take the item min width into consideration.

Reply:
enlighten me more pls, looks same to me

Reply:
### min width of these cards is 256px

in the demo that you provided before going to 3 cols the size of each item is 224px

![CleanShot 2024-02-13 at 09 08 30@2x](https://github.com/kodadot/nft-gallery/assets/44554284/075a4bed-5587-4e46-9db5-957a646c486b)

using dynamic grid , breaks only when 256px min width is not met anymore 

![CleanShot 2024-02-13 at 09 10 36@2x](https://github.com/kodadot/nft-gallery/assets/44554284/7cceedad-02ac-4a86-9b7e-afb3dfc01b2a)

Reply:
try to put this code snippet below your `v-for="(collection, index) in collections"`. and compare with your card, looks same to me

```vue
<div v-for="n in 12" :key="n" class="h-40 w-full bg-gray-800 text-white">
  {{ n }}
</div>
```

Reply:
oopps, wait

Reply:
aahh yes, the items are not even on the row with css